jupyterhub/templates/ingress.yaml

```diff
@@ -14,11 +14,12 @@ metadata:
     {{- end }}
 spec:
   rules:
+    {{- $root := . -}}
     {{- range $host := .Values.ingress.hosts }}
     - host: {{ $host }}
       http:
         paths:
-          - path: /
+          - path: {{ $root.Values.hub.baseUrl }}
```

Comment:
Can this just be .Values.hub.baseUrl? Does `range` modify `.`?

Reply:
`.` is modified in loops. It's poorly documented, it took me a while to figure out why `Values.hub.baseUrl` doesn't work: https://github.com/kubernetes/helm/issues/1311

Reply:
Wow that's crazy I haven't been bitten by this before haha! Thanks for finding it!